README.md

```diff
@@ -166,12 +166,19 @@ Configuration format is described in detail in the
   parameters values for instances;
 * `cartridge_bootstrap_vshard` (`boolean`, optional, default: `false`): boolean
   flag that indicates if vshard should be bootstrapped;
+* `cartridge_wait_buckets_discovery` (`boolean`, optional, default: `true`): boolean
+  flag that indicates if buckets should be discovered after vshard bootstrap;
```

Comment:
`buckets should be discovered` -> `we should wait for buckets discovering`.

Reply:
But routers should discovery buckets, not us)